integration-tests/src/test/java/com/pip/chatbot/integration/tests/tests/adminJokesEndpoints.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Jokes Endpoints
+
+  Background:
+    * url baseUrl
+    * header Authorization =  call read('basic-auth.js') {user: #(user), password: #(password)}
```

Comment:
call na callonce: https://stackoverflow.com/questions/56114446/how-can-we-read-a-json-file-only-once-for-entire-feature-file-in-karate